protocol/v1/qbft/instance/forks/genesis/fork.go

```diff
@@ -56,6 +57,7 @@ func (g *ForkGenesis) PrepareMsgValidationPipeline(share *beacon.Share, state *q
 		signedmsg.ValidateLambdas(identifier[:]),
 		signedmsg.ValidateSequenceNumber(state.GetHeight()),
 		signedmsg.AuthorizeMsg(share),
+		prepare.ValidatePrepareMsg(),
```

Comment:
lets rename it to be more generic as it just checking signers

Reply:
@nivBlox I will rename it but it will not be generic for other message types because spec uses the `prepare msg allows 1 signer` error message for this check (same for other types).

Reply:
done